app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -43,7 +43,7 @@ public Result getSelfParticipant() {
         UserSession session = getAuthenticatedSession();
         Study study = studyService.getStudy(session.getStudyIdentifier());
         
-        StudyParticipant participant = participantService.getParticipant(study, NO_CALLER_ROLES, session.getId());
+        StudyParticipant participant = participantService.getParticipant(study, session.getId(), false);
```

Comment:
For now I'm removing healthCode and encryptedHealthCode from the JSON just to be sure this happens. This can be replaced with a Jackson filter in a follow-on update.